src/components/ProductPage.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect, useState } from 'react';
+import './../css/product.css';
+
+
+interface ProductPageProps {
+    productId: string;
```

Comment:
Odd the tab spacing is different on this page. Again, having 'format on save' set should fix these discrepancies when you go back & save in this file.